modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -73,15 +73,29 @@ public String getIcon() {
 
 	public String getTitle() {
 		String title = _title;
-
 		HorizontalCard horizontalCard = getHorizontalCard();
 
 		if ((_title == null) && (horizontalCard != null)) {
 			title = horizontalCard.getTitle();
 		}
 
-		return LanguageUtil.get(
-			TagResourceBundleUtil.getResourceBundle(pageContext), title);
+		if (isLocalized()) {
+			title = LanguageUtil.get(
+				TagResourceBundleUtil.getResourceBundle(pageContext), title);
+		}
+
+		return title;
+	}
+
+	public boolean isLocalized() {
+		boolean localized = _localized;
+		HorizontalCard horizontalCard = getHorizontalCard();
+
+		if (horizontalCard != null) {
+			localized = horizontalCard.isLocalized();
+		}
+
+		return localized;
```

Comment:
@thienquach87 Sorry for bugging you so much, please make one last last change. We should implement here the standard logic for priority, where tag attribute has higher priority than model attribute. Something like:
```suggestion
	public boolean isLocalized() {
		boolean localized = _localized;
		
		HorizontalCard horizontalCard = getHorizontalCard();
		
		if ((_localized == null) && (horizontalCard != null)) {
			localized = horizontalCard.isLocalized();
		}
		
		return localized;
```

Reply:
@markocikos 

No worries, this is the first time I touch to clay components. I dug deeper into it faster when working with your. Based on your comments, It saved my time a lot!
I updated source to change the variable type from boolean to Boolean!
Thank you